lib/xml_builder.ex

```diff
@@ -261,8 +261,9 @@ defmodule XmlBuilder do
     map_intersperse(list, formatter.line_break(), &format(&1, level, options))
   end
 
-  defp format({nil, nil, name}, level, options) when is_bitstring(name),
-    do: [indent(level, options), to_string(name)]
+  defp format({nil, nil, content}, level, options) when is_bitstring(name) do
```

Comment:
What does `name` refer to in the guard?

Reply:
It doesn't refer to anything - classical case of last-second change without re-running test. Thanks for noticing, it's now fixed.